kolibri/core/assets/src/views/core-base.vue

```diff
@@ -18,8 +18,8 @@
       :navShown="navShown"
       :headerHeight="headerHeight"
       :width="navWidth"/>
-    <loading-spinner v-if="loading" class="loading-spinner-fixed"/>
     <div v-if="!loading" :style="contentStyle" class="content-container">
+      <loading-spinner v-if="loading" class="loading-spinner-aligned"/>
```

Comment:
I don't believe this will actually render, because it has `v-if="loading"` wrapped inside `v-if="!loading"` :)